src/static/static/home/yi-hack-v5/script/system.sh

```diff
@@ -227,7 +227,7 @@ fi
 
 ipc_multiplexer &
 
-mqttv4 &
+sleep 30 && mqttv4 &
```

Comment:
FYI 10 seconds didn't work, but 20 did on my system. I'm making it 30 to give it extra time for other models that I don't have to test with.